templates/CRM/Contact/Form/Task.tpl

```diff
@@ -12,7 +12,7 @@
 {if !empty($searchtype) && $searchtype eq 'ts_sel'}
 <div id="popupContainer">
   <div class="crm-block crm-form-block crm-search-form-block">
-    <table id="selectedRecords-{$group.id}" class="display crm-copy-fields crm-sortable">
+    <table id="selectedRecords-{if !empty($group)}{$group.id}{/if}" class="display crm-copy-fields crm-sortable">
```

Comment:
This is really weird. What is this dom id even used for?
Should we wrap the whole id property of this `<table>` inside the conditional? An id of `"selectedRecords-"` seems unlikely to be useful.

Reply:
@colemanw I didn't really look - I'm kinda at the point of just trying to make the notices go away & past the stage of thinking about the best way

Reply:
Fair enough. A question for another day.